modules/labs-react/layout/stories/Stack.stories.mdx

```diff
@@ -0,0 +1,154 @@
+import {ShouldWrapChildren} from './examples/ShouldWrapChildren';
+import {StackItems} from './examples/StackItems';
+import {NestedStacks} from './examples/NestedStacks';
+import {StackCard} from './examples/StackCard';
+import {HStackCards} from './examples/HStackCards';
+import {VStackCards} from './examples/VStackCards';
+
+<Meta title="Labs/Layout/Stack" component={Stack} />
+
+# Stack, HStack, and VStack
+
+[![LABS: Beta](https://img.shields.io/badge/LABS-beta-orange)](https://github.com/Workday/canvas-kit/tree/master/modules/labs-react/README.md)
+This component is work in progress and currently in pre-release.
+
+`Stack`, `HStack`, and `VStack` are higher-level layout components that provide an ergonomic API for
+building one-dimensional layouts with consistent spacing between elements. They're built on top of
+`Flex` and have access to all `FlexStyleProps` and `BoxProps`. `Stack` can align items in either a
+horizontal or vertical orientation. `HStack` and `VStack` will only align items in a horizontal or
+vertical orientation, respectively.
+
+## Stack
+
+`Stack` is a helpful component for creating layouts with equal spacing between elements. It can
+create horizontal or vertical stacks of elements.
+
+### Up & Running
+
+```tsx
+import {Stack, StackStyleProps} from '@workday/canvas-kit-labs-react/layout';
+```
+
+### What Stack Is & Isn't
+
+The most important distinction to make with `Stack` is that it is _not a grid._ Grid components are
+able to align items along two dimensions (rows and columns), and `Stack` is only built to support
+one dimension (rows _or_ columns). That said, you can nest horizontal and vertical `Stack`
+components to create grid-like layouts.
+
+### Setting Direction
+
+The direction of the child elements is set with the `flexDirection` prop. This prop supports four
+directions: `column`, `column-reverse`, `row`, and `row-reverse`. By default, it will set the
+direction to `row` as is consistent with the CSS Flexbox spec.
+
+### Setting Space
+
+The space between child elements is set with the `spacing` prop, which supports all space token and
+string values. `Stack` intentionally only sets space between elements and never outside. For
+example: a vertical stack of three elements would only have margin added to the top of the second
+and third child elements. Keeping space between elements makes building layouts more block-like and
+predictable.
+
+### Managing Child Elements
+
+#### Basic Usage
+
+`Stack` uses CSS pseudo-classes to apply margin to child elements. This makes it really simple to
+add child elements and get consistent spacing without much manual effort. Below is a basic example.
+Note that it does not add any space above or below the child elements. This helps your element fit
+cleanly within their containers.
+
+<ExampleCodeBlock code={BasicStack} />
+
+#### Using `shouldWrapChildren`
+
+Because of the increased specificity of `Stack`'s pseduo-classes, it may override the child element
```

Comment:
IMO this description isn't clear enough to indicate exactly when you should or shouldn't use this prop.

Reply:
Yep. I need to rethink this content a bit. I feel like there should be some general information below `Managing Child Elements` and then explain how each option works in these subsections.